.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,79 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+
+#v2
+
+# Workflow name
+name: "chromatic"
+
+on: push
```

Comment:
this would be changed to:

```
on:
  issue_comment:
    types: [created]
 ```
 
 before merging, but the action can only be tested with `on: push`